### ISSUE TITLE
eslint errors not showing up any more

### ISSUE THREAD

**Issue by Xeoncross** [AtomLinter/linter-eslint#1148]
## Issue Description

`linter-eslint` no longer reports errors or the output of eslint. I've been using `linter-eslint` previously for months. On save, no errors are reported though running eslint directly provides them.

 
```
$ node_modules/eslint/bin/eslint.js  lib/server.js 

  159:22  error  Missing trailing comma                           comma-dangle
  234:5   error  Expected space(s) after "if"                     keyword-spacing
  243:3   error  Expected space(s) after "if"                     keyword-spacing
  244:62  error  Missing semicolon                                semi
```

## Bug Checklist
- [X] Restart Atom
- [X] Re install `linter-eslint`
- [X] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
- [X] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

```
Atom version: 1.27.2
linter-eslint version: 8.4.1
ESLint version: 4.19.1
Hours since last Atom restart: 0.1
Platform: darwin
Using local project ESLint from: /Users/owner/go/src/github.com/xeoncross/server/node_modules/eslint
Current file's scopes: [
  "source.js",
  "meta.method-call.js",
  "meta.arguments.js",
  "meta.method-call.js",
  "meta.arguments.js",
  "meta.method-call.js",
  "meta.arguments.js",
  "meta.method-call.js",
  "meta.arguments.js",
  "meta.method-call.js",
  "entity.name.function.js"
]
linter-eslint configuration: {
  "disableWhenNoEslintConfig": false,
  "lintHtmlFiles": false,
  "useGlobalEslint": false,
  "showRuleIdInMessage": true,
  "eslintrcPath": "",
  "globalNodePath": "",
  "advancedLocalNodeModules": "",
  "eslintRulesDirs": [],
  "disableEslintIgnore": false,
  "disableFSCache": false,
  "fixOnSave": false,
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.babel",
    "source.js-semantic"
  ],
  "rulesToSilenceWhileTyping": [],
  "rulesToDisableWhileFixing": [],
  "ignoreFixableRulesWhileTyping": false
}
```

**Comment by Xeoncross** [comment]
Uninstalled everything, updated Atom to 1.28, and now it's working.

```
Atom version: 1.28.1
linter-eslint version: 8.4.1
ESLint version: 4.19.1
Hours since last Atom restart: 0.8
Platform: darwin
Using local project ESLint from: /Users/owner/go/src/github.com/server/node_modules/eslint
Current file's scopes: [
  "source.js.jsx",
  "meta",
  "meta.function.body",
  "meta.block.es",
  "meta.block.es",
  "punctuation.terminator.statement.js"
]
linter-eslint configuration: {
  "disableWhenNoEslintConfig": false,
  "rulesToSilenceWhileTyping": [
    "no-trailing-spaces",
    "eol-last"
  ],
  "lintHtmlFiles": false,
  "useGlobalEslint": false,
  "showRuleIdInMessage": true,
  "eslintrcPath": "",
  "globalNodePath": "",
  "advancedLocalNodeModules": "",
  "eslintRulesDirs": [],
  "disableEslintIgnore": false,
  "disableFSCache": false,
  "fixOnSave": false,
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.babel",
    "source.js-semantic"
  ],
  "rulesToDisableWhileFixing": [],
  "ignoreFixableRulesWhileTyping": false
}
```
